md/widgets.md

```diff
@@ -53,7 +53,7 @@ Each tab is a container widget.
 Use the **form container** to enable form validation for input widgets. The form container also exposes a local **$form** which is the AngularJS form object associated to the form container. In the Bonita Studio, if you define a contract for the task or process and click on the UI Designer icon, the generated form already contains a form container to embed default input widgets to collect the information that need to be passed to the contract to be verified. You can then define the validation you need on the input.  
 For example, you can bind the button's disabled property to _$form.$invalid_ to prevent user from clicking the button, until the form inputs are valid.
 
-### <a name="modal-widget"></a> Modal container widget
```

Comment:
if you change the name, you need to change the link in this page : https://github.com/bonitasoft/bonita-doc/blob/7.8/md/uid-modal-tutorial.md

Reply:
done.

Reply:
PR#621

Reply:
Yep. Done on 7.4 code. Here: PR#622